.github/workflows/extended.yml

```diff
@@ -41,13 +44,10 @@ jobs:
         with:
           submodules: true
           fetch-depth: 1
-      - name: Install Rust
-        run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          source $HOME/.cargo/env
-          rustup default stable
-      - name: Install Protobuf Compiler
-        run: sudo apt-get install -y protobuf-compiler
+      - name: Setup Rust toolchain
```

Comment:
This change simply changes the extended tests to use the standard setup builder job

Reply:
there was a good reason -- which is they ran out of diskspace otherwise